sql/chunk.sql

```diff
@@ -91,7 +91,8 @@ BEGIN
             range_start := (dimension_value / dimension_row.interval_length) * dimension_row.interval_length;
             range_end := range_start + dimension_row.interval_length;
         ELSE
-            range_end := (dimension_value / dimension_row.interval_length) * dimension_row.interval_length;
```

Comment:
Ideally, this is the kind of stuff we should have unit tests for, rather than regression tests.